### PR TITLE
[IMP] analytic: show all children accounts on plan

### PR DESCRIPTION
**Pull request by gawa-odoo**
On the smart button where you can see the accounts of the plan,
we want to be able to see all accounts of subplans too.
We also change the order of the analytic accounts, by plan's complete name then name.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tsb-odoo** [comment]
LGTM

File: addons/analytic/models/analytic_plan.py [hunk 1]
```diff
@@ -108,7 +120,7 @@ def action_view_analytical_accounts(self):
         result = {
             "type": "ir.actions.act_window",
             "res_model": "account.analytic.account",
-            "domain": [('id', 'in', self.account_ids.ids)],
+            "domain": [('id', 'in', self.all_children_account_ids.ids)],
```
Comment (william-andre) [inline addons/analytic/models/analytic_plan.py:123]: Let's add the field in `master`, we are changing enough things in stable like that

```suggestion
            "domain": [('id', 'in',(self + self.children_ids).account_ids.ids)],
```

File: addons/analytic/models/analytic_account.py [hunk 2]
```diff
@@ -10,7 +10,6 @@ class AccountAnalyticAccount(models.Model):
     _name = 'account.analytic.account'
     _inherit = ['mail.thread']
     _description = 'Analytic Account'
-    _order = 'code, name asc'
```
Comment (william-andre) [inline addons/analytic/models/analytic_account.py:13]: The message about this is quite unclear.
Why are you not ordering by default, but only on `view_account_analytic_account_list`?

File: addons/analytic/models/analytic_plan.py (hunk 1 above)
Comment (william-andre) [reply addons/analytic/models/analytic_plan.py:123]: Also not that this is only considering direct children. Isn't it possible to have more than one degree of parenthood?

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**